backend/capellacollab/sessions/hooks/jupyter.py

```diff
@@ -49,6 +57,22 @@ def _get_project_share_volume_mounts(
 
         for model in accessible_models_with_workspace_configuration:
             assert model.configuration
+            volume_name = (
+                "shared-workspace-" + model.configuration["workspace"]
+            )
+
+            if not operator.persistent_volume_exists(volume_name):
+                warnings.append(
+                    core_models.Message(
+                        err_code="JUPYTER_FILE_SHARE_VOLUME_NOT_FOUND",
+                        reason=(
+                            f"The Jupyter file-share volume for the model '{model.name}' in the project '{model.project.name}' couldn't be located. "
+                            "Please contact your system administrator or recreate the model (this will erase all data in the file-share)."
+                        ),
```

Comment:
This doesn't actually require any change, but seeing the core_models Message model after redoing all those exceptions made me think we could probably do a quick clean up of existing Messages too (right now all the fields are optional and pretty inconsistently applied).

Reply:
Sounds good, can you take care of it?